posts/fantastic-simprocs.md

```diff
@@ -60,7 +60,7 @@ Indeed, we will soon see an example of a simproc taking the place of infinitely
 
 # Examples of simprocs
 
-In this section, we exemplify four simprocs through the following use cases:
+In this section, we exemplify four simprocs that cover the following following use cases:
```

Comment:
I'm not following the following following following. I think it should be removed.